configuration/pih/scripts/visit/encounterTypeConfig.js

```diff
@@ -1107,6 +1107,31 @@ angular.module("encounterTypeConfig", [])
             ]
           }
         };
+        
+        // outpatientConsult
+        encounterTypes["92fd09b4-5335-4f7e-9f63-b2a663fd09a6"] = {
```

Comment:
This is tangential, I assume?

Reply:
Whoops! That should have gone in a while ago, with the Peru stuff. Opened https://github.com/PIH/openmrs-config-pihemr/pull/89